net/yaaw/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2014-2016 nanpuyue <nanpuyue@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yaaw
+PKG_VERSION:=2016-05-28
+PKG_RELEASE:=1
+PKG_MAINTAINER:=nanpuyue <nanpuyue@gmail.com>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/binux/yaaw.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=c28cde2252b707fe0661693f78c5e5e3beee09a1
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=http://www.gnu.org/licenses/lgpl.txt
```

Comment:
Quoting the [Contributing Guidelines](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain)

```
An optional PKG_LICENSE_FILES tag including the filenames of the license-files in the source-package. (E.g.: PKG_LICENSE_FILES:=COPYING)
```